app/src/main/java/org/fossasia/openevent/app/common/data/models/contract/IHeaderProvider.java

```diff
@@ -1,7 +1,8 @@
 package org.fossasia.openevent.app.common.data.models.contract;
 
 public interface IHeaderProvider {
-    String getHeader();
 
-    long getHeaderId();
+    String getHeader(int sortBy);
+
+    long getHeaderId(int sortBy);
```

Comment:
As these are common methods these shouldn't change

Reply:
@iamareebjamal custom sort could be implemented in all modules which have a list in them so isn't it better to modify these functions, otherwise i would have to create separate functions to get different headers which would then have to be implemented by other delegate classes unnecessarily.